wdl-parsing/WDLParsingFunction/pom.xml

```diff
@@ -0,0 +1,305 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.dockstore</groupId>
+    <artifactId>wdl-parser</artifactId>
+    <version>1.0</version>
+    <packaging>jar</packaging>
+    <name>WDL parser created for SAM CLI.</name>
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <cromwell.version>51</cromwell.version>
+    </properties>
+    <repositories>
+    <repository>
+        <id>artifactory.broadinstitute.org</id>
+        <name>artifactory.broadinstitute.org</name>
+        <url>https://artifactory.broadinstitute.org/artifactory/libs-release</url>
+        <releases>
+            <enabled>true</enabled>
+        </releases>
+        <snapshots>
+            <enabled>false</enabled>
+        </snapshots>
+    </repository>
+        <repository>
+            <id>jgit-repository</id>
+            <url>https://repo.eclipse.org/content/groups/releases/</url>
+        </repository>
+    </repositories>
+    <dependencies>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-lambda-java-core</artifactId>
+            <version>1.2.0</version>
+        </dependency>
+        <dependency>
+          <groupId>com.amazonaws</groupId>
+          <artifactId>aws-lambda-java-events</artifactId>
+          <version>3.1.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.broadinstitute</groupId>
+            <artifactId>wdl-draft3_2.12</artifactId>
+            <version>${cromwell.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.typelevel</groupId>
+                    <artifactId>cats-effect_2.12</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.typelevel</groupId>
+                    <artifactId>cats-core_2.12</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.typelevel</groupId>
+                    <artifactId>alleycats-core_2.12</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>eu.timepit</groupId>
+                    <artifactId>refined_2.12</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.11.1</version>
```

Comment:
Use `${jackson.version}` or `${jackson.major.version}`

Reply:
this lambda is completely separate from the dockstore code btw, not sure if that's obvious or not

Reply:
No, missed this. What was your reasoning on putting it in the Dockstore repo but not integrating it with the existing root pom or were you just thinking of that as a `TODO`?

Reply:
I mentioned during storytime that where to put the repo was still being decided upon, so it should be pretty obvious that I didn't integrated it with Dockstore yet.  I asked you privately where to put the code and you mentioned the main Dockstore repo, so I copied the code here.

Reply:
I think I also like the shared configuration with checkstyle, imports, findbugs. 
Plus we'd also have to tinker with dependencybot, snyk, lgtm so that they'd be aware of the separate project. 
That said, I think that can be a separate ticket when we get to it if you prefer that.

Reply:
Most definitely out of the scope of this PR. will add an issue to the epic for Snyk, LGTM, Dependabot

Reply:
Ok, then for now let's just move this pom down one level to `wdl-parsing` to make hooking this up to the main project easier later. L